jwt.py

```diff
@@ -12,8 +13,10 @@ def run(self, edit, text):
                 new_file = self.view.window().new_file()
                 new_file.run_command('append', {"characters": decoded_jwt})
                 new_file.assign_syntax(sublime.find_syntax_by_name("JSON")[0])
+                post_processing(new_file)
             except (UnicodeDecodeError, base64.binascii.Error):
                 sublime.error_message("Invalid JWT")
+            return
```

Comment:
Returning here I think makes it more consistent. If you paste the JWT to the prompt, it leaves other currently selected content alone.